tidy3d/web/api/container.py

```diff
@@ -27,7 +27,82 @@ class WebContainer(Tidy3dBaseModel, ABC):
 
 
 class Job(WebContainer):
-    """Interface for managing the task runs on the server."""
+    """
+    Interface for managing the running of a :class:`.Simulation` on server.
+
+    Notes
+    -----
+
+        This class provides a more convenient way to manage single simulations, mainly because it eliminates the need
+        for keeping track of the ``task_id`` and original :class:`.Simulation`.
+
+        We can get the cost estimate of running the task before actually running it. This prevents us from
+        accidentally running large jobs that we set up by mistake. The estimated cost is the maximum cost
+        corresponding to running all the time steps.
+
+        Another convenient thing about :class:`Job` objects is that they can be saved and loaded just like other
+        ``tidy3d`` components.
+
+    Examples
+    --------
+
+        Once you've created a ``job`` object using :class:`tidy3d.web.Job`, you can upload it to our servers with:
```

Comment:
the code examples deal with the regular tidy3d webapi, should be `job.upload()` etc.